cmd/auth.go

```
@@ -87,16 +99,14 @@ func readPassword() string {
 }
 
 // retrieves a basic access token from the wercker API
-func getAccessToken(username, password, url string) (string, error) {
-	creds := Credentials{
-		Username: username,
-		Password: password,
-		Scope:    "cli",
+func getAccessToken(username, password, sessionName, url string) (string, error) {
+	tokenRequest := &Request{
+		Name: sessionName,
 	}
 
-	b, err := json.Marshal(creds)
+	b, err := json.Marshal(tokenRequest)
 	if err != nil {
-		authLogger.WithField("Error", err).Debug("Unable to serialize credentials")
+		authLogger.WithField("Error", err).Debug("Unable to serialize the session name")
```

Changing this to `the session name` doesn't seem accurate to me. It is the object which isn't able to be marhsalled. We could just ignore the error since there is no way it can be triggered.